### PR TITLE
fix(text-input): add missing disabled color token

### LINKED ISSUE #2028: Disabled input field text missing disabled color token
This is a screenshot of text within a disabled field:

image

The text does not have the correct color token applied

### PR DESCRIPTION
**Pull request by emyarod**
Closes #2028 

Related: https://github.com/IBM/carbon-components-react/issues/1989

This PR adds the disabled color value to text contained within disabled text input fields

### CODE REVIEW

File: src/components/text-input/_text-input.scss [hunk 1]
```diff
@@ -208,6 +208,7 @@
     outline: none;
     background-color: $disabled-01;
     border-bottom: 1px solid transparent;
+    // color: $disabled-02;
```
Comment (asudoh) [inline src/components/text-input/_text-input.scss]: Did you mean to uncomment this @emyarod?

Comment (emyarod) [reply src/components/text-input/_text-input.scss]: yes, should be fixed now

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @emyarod!

**Review by lovemecomputer — APPROVED** [review]
cool!

**Pull request merged**